src/components/building-information/link-element.component.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { StyleSheet, Text, View, Linking } from "react-native";
+import { LinkItem } from "../../types/main";
+import { LINK_COLOR } from "../../styles";
+
+interface IProps {
+  linkItem: LinkItem;
```

Comment:
Can I ask for the name 'linkItem' to be changed to hyperlinkItem, urlLinkItem or something similar? linkItem can be interpreted as linking between two items